src/NuGetTrends.Scheduler/NuGetTrends.Scheduler.csproj

```diff
@@ -11,20 +11,20 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="MessagePack" Version="2.2.44-rc" />
+    <PackageReference Include="MessagePack" Version="2.2.85" />
     <PackageReference Include="NuGet.Client" Version="4.3.0-beta1-2418" />
-    <PackageReference Include="RabbitMQ.Client" Version="6.2.1" />
-    <PackageReference Include="Hangfire.AspNetCore" Version="1.7.13" />
+    <PackageReference Include="RabbitMQ.Client" Version="6.2.2" />
+    <PackageReference Include="Hangfire.AspNetCore" Version="1.7.24" />
     <PackageReference Include="Hangfire.MemoryStorage" Version="1.7.0" />
-    <PackageReference Include="Sentry.AspNetCore" Version="3.3.1" />
-    <PackageReference Include="Sentry.Serilog" Version="3.3.1" />
-    <PackageReference Include="Serilog.AspNetCore" Version="3.4.1-dev-00180" />
-    <!-- 3.2.0-dev-00244 failed to load logz.io sink: https://github.com/serilog/serilog-settings-configuration/issues/237 -->
-    <PackageReference Include="Serilog.Settings.Configuration" Version="3.1.0" />
-    <PackageReference Include="Serilog.Sinks.Console" Version="4.0.0-dev-00839" />
-    <PackageReference Include="Serilog.Sinks.Logz.Io" Version="2.2.2" />
-    <PackageReference Include="Serilog.Enrichers.Environment" Version="2.2.0-dev-00761" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="5.0.0">
+    <PackageReference Include="Sentry.AspNetCore" Version="3.8.2" />
+    <PackageReference Include="Sentry.Serilog" Version="3.8.2" />
+    <PackageReference Include="Serilog.AspNetCore" Version="4.1.0" />
+    <PackageReference Include="Serilog.Settings.Configuration" Version="3.2.0-dev-00272" />
+    <PackageReference Include="Serilog.Sinks.Console" Version="4.0.1-dev-00874" />
+    <!-- 3.0.1 is unoptimized: https://github.com/mantasaudickas/serilog-sinks-logz-io/issues/20 -->
+    <PackageReference Include="Serilog.Sinks.Logz.Io" Version="3.0.0" />
```

Comment:
Tried the latest but unoptimized: https://github.com/mantasaudickas/serilog-sinks-logz-io/issues/20